packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -64,13 +64,15 @@ export default function ClassifierContainer({
   onSubjectReset = DEFAULT_HANDLER,
   onToggleFavourite = DEFAULT_HANDLER,
   project,
-  showTutorial=false,
+  showTutorial = false,
   subjectID,
   subjectSetID,
   workflowID
 }) {
   const storeEnvironment = { authClient, client }
   const { user, upp, projectRoles, userHasLoaded } = usePanoptesUserSession({ authClient, projectID: project?.id })
+
+  /** When project status is private, user must have one of the following roles to view its workflows */
```

Comment:
```suggestion
  /** When a workflow is inactive, user must have one of the following roles to view its workflows */
```

Reply:
When a project status is private, the NextJS app can't fetch it from Panoptes, so the page returns 404.

Reply:
We do support auth'ed requests for private projects in the dev classifier:
https://github.com/zooniverse/front-end-monorepo/blob/615d6af1e5e01e246200d43a7a1dd975e2b19a27/packages/lib-classifier/dev/components/App/App.js#L82-L89